### DISCUSSION TITLE
打包后报错

### DISCUSSION THREAD

**Discussion by OoOoYY** [zouyaoji/vue-cesium#239] [General]
![image](https://user-images.githubusercontent.com/65013993/182270206-da8c5aa2-f3d0-45be-9492-82caa229ec15.png)

**Comment by OoOoYY** [comment]
"vue": "3.2.4"

**Comment by OoOoYY** [comment]
"vue-cesium": "^3.0.15"

**Comment by OoOoYY** [comment]
![image](https://user-images.githubusercontent.com/65013993/182274920-1312d9f1-2f80-436e-9e4a-a4604059a5ed.png)
屏蔽掉后显示正常

**Comment by zouyaoji** [comment]
![image](https://user-images.githubusercontent.com/24430116/182289577-bc5426b7-38b7-4dcb-8b74-1b353ee8a337.png)

是你自己没定义这个变量哈。
